unit-tests-java8/src/test/java/org/eclipse/collections/test/RichIterableTestCase.java

```diff
@@ -1590,13 +1591,25 @@ default void RichIterable_makeString_appendString()
                 iterable.makeString());
 
         assertEquals(
+                iterable.makeString(),
+                iterable.reduceInPlace(makeString()));
+
+        assertEquals(
                 "4/4/4/4/3/3/3/2/2/1",
                 iterable.makeString("/"));
 
         assertEquals(
+                iterable.makeString("/"),
+                iterable.reduceInPlace(makeString("/")));
+
+        assertEquals(
                 "[4/4/4/4/3/3/3/2/2/1]",
                 iterable.makeString("[", "/", "]"));
 
+        assertEquals(
+                iterable.makeString("[", "/", "]"),
+                iterable.reduceInPlace(makeString("[", "/", "]")));
+
         StringBuilder builder1 = new StringBuilder();
```

Comment:
RichIterable_makeString_appendString is overridden in subclasses of RichIterableTestCase. I'd prefer if this were in a new test method, but it could stay in the same test as long as you also modify the overrides.